CODEOWNERS

```diff
@@ -146,6 +146,8 @@
 /drivers/dma/*dw*                         @tbursztyka
 /drivers/dma/*sam0*                       @Sizurka
 /drivers/dma/dma_stm32*                   @cybertale
+/drivers/dma/dma_mcux*                    @MaureenHelm
+/drivers/dma/Kconfig.imx_rt               @MaureenHelm
```

Comment:
Once you have renamed the files to `Kconfig.mcux_edma` and `dma_mcux_edma.c`, you shouldn't need to add these lines anymore. They're covered by `/drivers/*/*mcux*`

Reply:
ok